core/src/templates/components/quote/quote.twig

```diff
@@ -1,8 +1,8 @@
-<div class="su-quote">
+<article class="su-quote">
```

Comment:
I disagree with changing the quote to use <article>. I don't think quotes are stand alone content - I think they almost always need to be taken in context.

Reply:
I can buy that argument. Would `<section>` be more appropriate then?

Reply:
<section> sounds good to me. @JBCSU ?

Reply:
Another idea is to use `<figure>` - I know on stanford.edu, the event cards are figure elements. I looked up what it really means and it says [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure):
"The HTML <figure> (Figure With Optional Caption) element represents self-contained content, potentially with an optional caption, which is specified using the (<figcaption>) element. The figure, its caption, and its contents are referenced as a single unit." 
"Usually a `<figure>` is an image, illustration, diagram, code snippet, etc., that is referenced in the main flow of a document, but that can be moved to another part of the document or to an appendix without affecting the main flow."

Reply:
@cjwest Actually, the more I think about this, `<figure>` actually seems like a good choice here. I'm working on the WordPress template, and their Gutenberg WYSIWYG editor also output a `<figure>` element for their fancy quote component. They have a `<blockquote>` (for the quote and cite portion) inside a `<figure>`.

Reply:
@yvonnetangsu,
FYI, I am excited we are exploring this topic!
So you're suggesting that for the quote we use `<figure>`?
``` 
<figure class="su-quote">
```
I'm not sure it's fitting semantically. Also, I don't think that an image should be required for a quote so having a figure without a image would be weird. The body or quote, however, should be required.
 I'd welcome other opinions on this.

Reply:
@cjwest , apparently `<figure>` isn't only for elements with an image, for example, a lot of article comments are individually wrapped within a `<figure>` tag also. WordPress' quote block and video blocks also use `<figure>` and there are no images involved.

Reply:
Check out Smashing Magazine's individual comments for example:
https://www.smashingmagazine.com/2018/06/bem-for-beginners/

I'm glad we're having a discussion on this as well! :)

Reply:
Here's an example where they combine a figure with a block quote.

https://github.com/hail2u/html-best-practices#dont-include-attribution-directly-in-blockquote-element

What are your thoughts on this:
```
<figure>
  <img class="su-quote__img" src="{{ src }}" alt="{{ alt }}">
   <p class="su-quote__heading">{{ name }}</p>
  <blockquote>
    <p class="su-quote__quote">"{{ text }}"</p>
  </blockquote>
  <figcaptionclass="su-quote__bio">{{ bio }}</figcaption>
</figure>
```

Reply:
I need a little more time to think about this but I'm going on vacation tomorrow 😄 . I know I don't like the one with the nested `figure` though - don't think that's allowed.

Reply:
Actually, you can nest '<figure>`